src/main/java/org/apache/mesos/scheduler/plan/Block.java

```diff
@@ -14,43 +13,13 @@
  * <p>
  * See {@Stage} docs for more background.
  */
-public interface Block extends Completable {
-
-    /**
-     * Utility method which maps the provided {@link Block} to its corresponding {@link Status}.
-     */
-    static Status getStatus(Block block) {
-        // these should all be mutually exclusive, ordering doesn't matter.
-        if (block.isPending()) {
-            return Status.PENDING;
-        } else if (block.isInProgress()) {
-            return Status.IN_PROGRESS;
-        } else if (block.isComplete()) {
-            return Status.COMPLETE;
-        }
-        return Status.ERROR;
-    }
-
+public interface Block extends Element {
```

Comment:
Sorta surprised this works -- doesn't it complain that Element isn't being passed a template type?